src/main/views/course/manage.html

```diff
@@ -67,16 +69,18 @@ <h3>Artifact 1: {{name}}</h3>
 						</td>
 						{{/evaluation}}
 						<td>
-							<input name="student-1-submission" type="file"></input>
+							<input name="student-1-submission" type="file", {{#read_only}}disabled{{/read_only}}></input>
 							attach <i class="material-icons">attachment</i>
 						</td>
 					</tr>
 					{{/evaluations}}
 				</table>
-				<input type="submit" />
+				<input type="submit", {{#read_only}}disabled{{/read_only}}>
```

Comment:
No comma needed after `attr="something"`. Same on lines 61, 72. You don't have to change it.